Dockerfile

```diff
@@ -22,6 +22,20 @@ ENV GOPATH=/go \
     GOPROXY=https://proxy.golang.org \
     PATH=$PATH:/go/bin
 
+# confluent-kafka-go is not built for aarch64
+# https://github.com/confluentinc/confluent-kafka-go/issues/576#issuecomment-1009766473
+RUN if [ "$(uname -m)" == "aarch64" ] ; then \
+        DIR=$(pwd) ; \
+        # build librdkafka from source for aarch64
+        git clone --branch v1.6.2 https://github.com/edenhill/librdkafka.git && cd librdkafka ; \
+        dnf install -y gcc-c++ make cyrus-sasl-devel cyrus-sasl-lib cyrus-sasl-gssapi libcurl-devel ; \
+        ./configure --install-deps ; \
+        make ; \
+        make install ; \
+        ln -s /usr/local/lib/librdkafka.so.1 /usr/lib64/librdkafka.so.1 ; \
```

Comment:
I see it also builds static library in librdkafka/src/librdkafka.a, can't you just add it to module path so the rest of the files don't have to deal with dynamic linking magic?

Reply:
yeah, I saw that too but some logs suggested like librdkafka.a doesn't contain everything (if thats possible). Dynamic linking is what confluent-kafka-go readme tells us to do, but I can try to use static library